Google Scholar.js

```diff
@@ -866,6 +886,8 @@ ItemFactory.prototype.getTitle = function () {
 	// Citelet parsing, step (4) [optional]
 	if (this.commaSplit) {
 		this.v.title = this.commaSplit.join(", ");
+	} else {
+		throw new Error("Parsing failed for case item \"" + this.citelet + "\"");
 	}
```

Comment:
Understood. I'll go back and test with the original translator code, and pin down the cause.